locallib.php

```diff
@@ -660,6 +660,55 @@ function zoom_get_nonusers_from_alternativehosts(array $alternativehosts) {
     }
 }
 
+/**
+ * Get the unavailability note based on the Zoom plugin configuration.
+ *
+ * @param object $zoom The Zoom meeting object.
+ * @param bool|null $finished The function needs to know if the meeting is already finished.
+ *                       You can provide this information, if already available, to the function.
+ *                       Otherwise it will determine it with a small overhead.
+ *
+ * @return string The unavailability note.
+ */
+function zoom_get_unavailability_note(object $zoom, $finished = null) {
+    // Get config.
+    $config = get_config('zoom');
+
+    // Get the plain unavailable string.
+    $strunavailable = get_string('unavailable', 'mod_zoom');
+
+    // If this is a recurring meeting, just use the plain unavailable string.
+    if ($zoom->recurring === true) {
```

Comment:
Sorry! It looks like this is coming from the database and may not be a boolean, thus type coercion is probably acceptable. This format mirrors other pre-existing code:
```suggestion
    if ($zoom->recurring) {
```
Alternatively, we could be explicit that we expect type coercion:
```suggestion
    if (!empty($zoom->recurring)) {
```